osmosis-1/osmosis.zone_assets.json

```diff
@@ -716,6 +716,27 @@
         }
       ]
     },
+    {
+      "chain_name": "gravitybridge",
+      "base_denom": "gravity0x45804880De22913dAFE09f4980848ECE6EcbAf78",
+      "path": "transfer/channel-144/gravity0x45804880De22913dAFE09f4980848ECE6EcbAf78",
+      "osmosis_verified": false,
+      "override_properties": {
+        "name": "Paxos Gold (Gravity Bridge)",
+        "symbol": "PAXG.grv",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/paxg.grv.svg"
```

Comment:
This file doesn't exist. If you can register PAXG to /_non-cosmos/ethereum/, I can create it in the same style.

Reply:
added paxg.grv.svg in same style of other assets in https://github.com/cosmos/chain-registry/commit/40c02237f8803fd65e0f1f145d6093b920e74937